cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -215,7 +225,11 @@ def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
             4.Change number of CPU sockets and amount of memory, save and submit
             5. Check the count in VM details page
     """
-    pass
+    orig_config = small_vm.configuration.copy()
+    new_config = prepare_new_config(orig_config, change_type)
+
+    # Apply new config
+    reconfigure_vm(small_vm, new_config)
```

Comment:
Any assertions that we can do to check if VM is reconfigured ?

Reply:
added assertions